apps/web/test/lib/next-config.test.ts

```diff
@@ -80,7 +80,8 @@ describe("next.config.js - Org Rewrite", () => {
 
   describe("Rewrite", () => {
     it("booking pages", () => {
-      expect(orgUserTypeRouteMatch("/user/type")?.params).toContain({
+      console.log(orgUserTypeRouteMatch("/user/type").params);
```

Comment:
Still need this?

Reply:
Oh no good spot!

Reply:
Fixed https://github.com/calcom/cal.com/pull/16676/commits/adb09fbb633be1797c907a36933d288b8bc64e58